packages/publisher/base-static/test/StaticPublisher_spec.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from 'chai';
+
+import { PublisherStatic, StaticPublisherConfig } from '../src/PublisherStatic';
+
+class PublisherImpl extends PublisherStatic<StaticPublisherConfig> {
+  defaultPlatforms = [];
+
+  name = 'test';
+
+  public exposedKeyForArtifact = this.keyForArtifact;
+}
+
+describe('PublisherStatic', () => {
+  describe('keyForArtifact', () => {
```

Comment:
I respect the optimism that the spec will see more `describe` calls

Reply:
One day 🤞